packages/react-native/sdks/hermes-engine/utils/replace_hermes_version.js

```diff
@@ -62,7 +62,7 @@ function replaceHermesConfiguration(configuration, version, podsRoot) {
   fs.mkdirSync(finalLocation, {recursive: true});
 
   console.log('Extracting the tarball');
-  execSync(`tar -xf ${tarballURLPath} -C ${finalLocation}`);
+  execSync(`tar -xzf ${tarballURLPath} -C ${finalLocation}`);
```

Comment:
why is this needed?

Reply:
Out of curiosity, I went browsing through the changelogs, seems like the behavior has changed since 2004-12-20  with release 1.15 of tar 😅